test/test_owners.csv

```diff
@@ -766,7 +766,7 @@ k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults,fgrzadkowski,0,
 k8s.io/kubernetes/pkg/scheduler/api/validation,fgrzadkowski,0,
 k8s.io/kubernetes/pkg/scheduler/core,madhusudancs,1,
 k8s.io/kubernetes/pkg/scheduler/factory,fgrzadkowski,0,
-k8s.io/kubernetes/pkg/scheduler/schedulercache,fgrzadkowski,0,
+k8s.io/kubernetes/pkg/scheduler/cache,fgrzadkowski,0,
```

Comment:
I think this file is expected to be sorted alphabetically.

Reply:
DONE!